client/src/components/popup/PopupModal.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './PopupModal.scss';
+import XMark from '../../assets/misc/xmark-solid-white.svg';
+
+const PopupModal = ({
```

Comment:
Tapping the background, should close the popup
You can use something like `useWrapperRef.jsx`
It was used in the `Dropdown` component